### PR TITLE
feat(theming): update components with css custom variables - CRM theme

### PR DESCRIPTION
**Pull request by srivalli-sivaramasankaran**
- Button - Primary & Secondary
- Label
- Input

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] My commits have standard messages as mentioned in [Contributing Guidelines](./../blob/next/CONTRIBUTING.md)

### CODE REVIEW

File: packages/crayons-core/src/components/button/button.scss [hunk 1]
```diff
@@ -7,27 +7,75 @@
...
+  @prop --fw-button-secondary-border: The border color of the secondary button.
+  @prop --fw-button-secondary-border-active: The border color of the secondary button on active state.
+  @prop --fw-button-secondary-bg: The background color of the secondary button.
+  @prop --fw-button-secondary-bg-dark: The background color of the secondary dark button.
+  @prop --fw-button-secondary-color: The text color of the secondary button.
+  @prop --fw-button-secondary-disabled-color: The text color of the secondary disabled button.
+  @prop --fw-button-secondary-disabled-border: The border color of the secondary disabled button.
+  @prop --fw-button-secondary-disabled-bg: The background color of the secondary disabled button.
+  @prop --fw-button-active-box-shadow: Box shadow of active button.
+  @prop --fw-spinner-color-button-secondary: Spinner color of the secondary button.
```
Comment (kishore-kumar-E3682) [inline packages/crayons-core/src/components/button/button.scss:30]: Can`--fw-spinner-color-button-secondary` be renamed to `--fw-button-secondary-spinner-color`

File: packages/crayons-core/src/index.html [hunk 2]
```diff
@@ -10,9 +10,364 @@
...
+      --fw-button-primary-disabled-bg: #A2C0DE;
+      --fw-button-primary-disabled-bg-dark: #83A6C8;
+      --fw-icon-size: 14px;
+      --fw-button-secondary-color: #333;
+      --fw-button-secondary-border: #CFD7DF;
+      --fw-button-secondary-border-active: #CFD7DF;
+      --fw-button-secondary-bg: #FFFFFF;
+      --fw-button-secondary-bg-dark: #F5F7F9;
+      --fw-button-secondary-disabled-color: #999999;
+      --fw-button-secondary-disabled-border: #CFD7DF;
```
Comment (kishore-kumar-E3682) [inline packages/crayons-core/src/index.html:80]: The value of this css variable is same as that of the default value. So can this be removed?

File: packages/crayons-core/src/components/button/button.scss (hunk 1 above)
Comment (srivalli-sivaramasankaran) [reply packages/crayons-core/src/components/button/button.scss]: Fixed.

File: packages/crayons-styles/package.json [hunk 3]
```diff
@@ -18,6 +18,7 @@
   "main": "dist/index.js",
   "types": "dist/src/index.d.ts",
   "type": "module",
+  "files": ["dist", "css"],
```
Comment (arvindanta) [inline packages/crayons-styles/package.json:21]: Hi, do we need `css` directory here ?. I suppose, we have a build script at the root of package.json `copy:themes` that copies the css folder from here and puts it inside `crayons-core/css`.

File: packages/crayons-styles/scripts/convertTokens.js [hunk 4]
```diff
@@ -5,25 +5,32 @@ import { convertTokenToString } from '../src/utils.js';
 const tokenOutDir = './tokens/';
 const cssOutDir = './css/';
 const tokensDir = './design-tokens/';
+const globalFileName = 'crayons.json';
 
 function readFiles(rootPath) {
```
Comment (arvindanta) [inline packages/crayons-styles/scripts/convertTokens.js:10]: can we split this function in to 2 , one for readingFiles and one for parsing the contents of the file to form combinedTokens and return that value

**Review by arvindanta — CHANGES_REQUESTED** [review]

File: packages/crayons-styles/package.json (hunk 3 above)
Comment (srivalli-sivaramasankaran) [reply packages/crayons-styles/package.json:21]: Fixed.

File: packages/crayons-styles/scripts/convertTokens.js (hunk 4 above)
Comment (srivalli-sivaramasankaran) [reply packages/crayons-styles/scripts/convertTokens.js]: As discussed, renamed the function name.

**Review by arvindanta — APPROVED** [review]

**Pull request merged**